src/player.js

```diff
@@ -38,11 +59,24 @@ Player.prototype.makeDeposit = function(amount) {
 };
 
 Player.prototype.moveTo = function(position) {
```

Comment:
should this have 'collectGo' as an argument? (ie do not pass go, do not collection 200..)